### ISSUE TITLE
Collection Visibility

### ISSUE THREAD

**Issue by KelleyCoda** [MukurtuCMS/mukurtucms#36]
From Karuk: I would like for it to be that when you have a collection, that you should not be able to see the collection (on the list at /collections) if you cannot access any of the digital heritage items within.

**Comment by steve-taylor-wsu** [comment]
Agreed, I'll add that we'll probably still show the collection to users that have edit rights to that collection no matter what.
